tests/lib/get_snap_sha.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+from urllib import request
+from pprint import pprint
+import json
+import sys
+
+
+if len(sys.argv) != 4:
+	print("Number of parameters extected is 2: snap, channel and architecture")
```

Comment:
Two or three?

How about:
```
import argparse

parser = argparse.ArgumentParser()
parser.add_argument("snap")
parser.add_argument("channel")
parser.add_argument("arch")
opts = parser.parse_args()
```

Then you can use `opts.channel` instead of `channel`. It's just as long but the error messages are nicer and parsing is more complete.

Reply:
This is not needed anymore, this file has been removed. Thanks for reviewing that :)